hw-model/test-fw/mcu_hitless_update_flow.rs

```diff
@@ -0,0 +1,78 @@
+// Licensed under the Apache-2.0 license
+
+//! A very simple program that responds to the mailbox.
```

Comment:
Is this comment out of date?

Also, it might be worth including a comment that this test is meant to be run from the MCU sw repo